Packs/FeedIntel471/Integrations/Intel471Actors/Intel471Actors.py

```diff
@@ -53,14 +58,15 @@ def custom_build_iterator(client: Client, feed: Dict, limit: int = 0, **kwargs)
     result: List[Dict] = []
     should_continue = True
     total_count = 0
+    request_headers = {'user-agent': USER_AGENT}
```

Comment:
Let's remove this line and define the `headers` as a parameter in the main function  and pass it to the feed_main function instead.
And in the main function #L166
```python
params['headers'] = {'user-agent': USER_AGENT}

```
_**Relevant for all the feeds**_